### DISCUSSION TITLE
apprentissage

### DISCUSSION THREAD

**Discussion by FabienMich** [community/community#106004] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Salut, au fait je suis intéressé par l'apprentissage de ce domaine, stp initie-moi je veux devenir un hacker c'est mon rêve

**Comment by queenofcorgis** [comment]
Hi @FabienMich , we're happy you're here 😀

However, we currently do not allow job listings or job seeking on the Community on Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.
